build/tsconfig.docs.json

```diff
@@ -7,7 +7,9 @@
       "docs/*": ["docs/*"],
       "src/*": ["packages/react/src/*"],
       "test/*": ["packages/react/test/*"]
-    }
+    },
+    "typeRoots": ["../types", "../node_modules/@types"],
```

Comment:
Sorry this might be obvious/very unrelated but is this basically allowing custom types for the documentation site but removing it for the actual component code?

Reply:
It's okay, it's good to have more people familiar with this stuff! What you said is generally correct. I took out the custom `typeRoots` by default (from `tsconfig.common.json`, which the other configs inherit from) because not all the projects use them. Then I added back the `typeRoots` to only the projects where they're needed.

The other reason for removing `typeRoots` from `tsconfig.common.json` (and this is more detail than you typically need to know) is that if other projects extend from it using the package name (`"extends": "@fluentui/internal-tooling/tsconfig.common.json"`) instead of a relative path, the `typeRoots` will get resolved relative to `node_modules/@fluentui/internal-tooling/tsconfig.common.json`. This is bad because it makes a lot of types get resolved incorrectly...it took me awhile to figure out what happened. :(

Reply:
Ah ok - thank you for the explanation!